vuepress/docs/next/tutorials/ecr/publish-simple-bundle.md

```diff
@@ -81,11 +81,11 @@ ent prj pbs-publish
 ```
 On subsequent iterations you can run just this command again to quickly push a new version of your bundle to git.
 
-5. You can now create the custom resource for your bundle and apply it to Kubernetes in one step. You should modify the target namespace parameter (`-n`) if you changed it from the default.
+5. You can now register the bundle into the ECR registry in one step.
```

Comment:
Same, change register to deploy, and maybe just say ECR instead of ECR registry.